File: backend/src/main/java/cha_revelacao/dto/response/UsuarioResponse.java
package cha_revelacao.dto.response;

import cha_revelacao.model.Usuario;
import lombok.Data;

import java.time.LocalDateTime;

@Data
public class UsuarioResponse {
    private Long id;
    private String nome;
    private String email;
    private String fotoPerfil;
    private Usuario.TipoUsuario tipo;
    private Boolean ativo;
    private LocalDateTime criadoEm;

    public Long getId() {
        return id;
    }
}

File: backend/src/DTOs/Responses/UsuarioResponse.cs
namespace MemuVie.Evento.DTOs.Responses;

using MemuVie.Evento.Models;

public class UsuarioResponse
{
    public long Id { get; set; }
    public string Nome { get; set; } = string.Empty;
    public string Email { get; set; } = string.Empty;
    public string? FotoPerfil { get; set; }
    public UserType Tipo { get; set; }
    public bool Ativo { get; set; }
    public DateTime CriadoEm { get; set; }
}
